index.html

```diff
@@ -0,0 +1,141 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">My Logo</div>
```

Comment:
- [ ] You did a great job designing the toolbar of your portfolio project! 🍾 But, it would be a good idea to wrap your logo with an `<a>` tag. By convention, when the user clicks (or taps) on the page's logo, he navigates to the main page. 📱  So including it would improve the semantics of your page and also the user experience. Please, use it! 🛠️

Reply:
@anagudelogu

Reply:
@Hi @anagudelogu, thank you for the suggestion. I've now wrapped my logo with a <a> tag.